Userland/Utilities/ping.cpp

```diff
@@ -180,6 +180,15 @@ int main(int argc, char** argv)
                 return 1;
             }
 
+            char ihl = -1;
```

Comment:
Since this is our own variable we can give it a longer name.
I personally didn't know what 'ihl' stands for and perhaps the next person reading this code won't either.